components/sendgrid/actions/validate-email/validate-email.js

```diff
@@ -0,0 +1,49 @@
+const validate = require("validate.js");
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "sendgrid-validate-email",
+  name: "Validate Email",
+  description:
+    "Validates an email address. This action requires a Sendgrid's Pro or Premier plan.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    email: {
+      type: "string",
+      label: "Email",
+      description: "The email that you want to validate.",
+    },
+    source: {
+      type: "string",
+      label: "Source",
+      description:
+        "An optional indicator of the email address's source. You may include this if you are capturing email addresses from multiple locations.",
+      optional: true,
+    },
+  },
+  methods: {
+    ...common.methods,
+  },
+  async run() {
```

Comment:
Include the `$` variable.
```suggestion
  async run({ $ }) {
```